conjure-java-core/src/main/java/com/palantir/conjure/java/types/BeanBuilderGenerator.java

```diff
@@ -488,9 +492,11 @@ private boolean isPrimitiveOptional(OptionalType optionalType) {
                         optionalType.getItemType().accept(TypeVisitor.PRIMITIVE).get());
     }
 
-    private MethodSpec createItemSetter(EnrichedField enriched, Type itemType, boolean override) {
+    private MethodSpec createItemSetter(
+            EnrichedField enriched, Type itemType, boolean override, Optional<LogSafety> safety) {
         FieldSpec field = enriched.poetSpec();
-        return BeanBuilderAuxiliarySettersUtils.createItemSetterBuilder(enriched, itemType, typeMapper, builderClass)
+        return BeanBuilderAuxiliarySettersUtils.createItemSetterBuilder(
```

Comment:
previously, item setters for collection types didn't have safety annotations, even for primitive collections. For example, a field of type `List<boolean>` would generate the following setter:

`        public Builder listBoolean(String listBoolean) {
            checkNotBuilt();
            this.listBoolean.add(listBoolean);
            return this;
        }`

Now, both primitive and external import collections generate item setters that look like:

`        public Builder listBoolean(@Safe String listBoolean) {
            checkNotBuilt();
            this.listBoolean.add(listBoolean);
            return this;
        }`